docs/deep-dives/mobile/nimbus-cli/20-getting-started.mdx

```diff
@@ -86,6 +103,8 @@ You can use the `--preserve-targeting` and `--preserve-bucketing` options to pre
 
 Enrolled experiments should persist across restarts of the app, and fetching of new experiments is disabled.
 
+Occassionally, you may want to get the experiment from Remote Settings rather than Experimenter. You can do this by adding the `--use-rs` flag.
```

Comment:
```suggestion
:::tip
Occassionally, you may want to get the experiment from Remote Settings rather than Experimenter. You can do this by adding the `--use-rs` flag.
:::
```